services/publisher.py

```
@@ -0,0 +1,175 @@
+                with httpx.Client(timeout=httpx.Timeout(connect=10.0, read=60.0)) as client:
+                    response = client.put(
+                        url,
+                        content=f.read(),
+                        headers={
+                            "Authorization": f"Bearer {self.cloudflare_api_token}",
+                            "Content-Type": "video/mp4"
+                        }
+                    )
+                    response.raise_for_status()
+                    
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)